include/blogWithComments_functions.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+$Error = array();
+
+
+function RepopulateComments($info){
+		global $Error;
+	if(isset($_REQUEST['submitComment']) && (sizeof($Error)>0) && isset($_REQUEST[$info])){
+		echo "<input type='text' name=$info value = $_REQUEST[$info] >";
+	}
+	else {
+		echo "<input type='text' name=$info>";
+	}
+}
+
+
+function SubmitComment($Name, $Email, $Comment){
+		$result = dbQuery("
+			INSERT INTO BlogComments(Name, Email, Comment)
+			VALUES('$Name', '$Email', '$Comment')
+		")->fetch();
+}
+
+
+if(isset($_REQUEST['submitComment'])){
+	Validate();
+	if((sizeof($Error)) == 0){
+		SubmitComment(
+				$_REQUEST['NameField'],
+				$_REQUEST['EmailField'],
+				$_REQUEST['CommentField']
+		);
+	}
+	if(sizeof($Error) > 0){
+		echo "<p class='ErrorMessage'>In order to submit a comment, please fill out all the fields!</p>";
+	}
+}
+
+
+function Validate(){
+	global $Error;
+	if (trim($_REQUEST['NameField'] == '') or (strlen($_REQUEST['NameField']) < 3)){
+		$Error['NameField']= "required";
+	}
+	if (trim($_REQUEST['EmailField']) == ''){
+		$Error['EmailField']= "required";
+	}
+	if (trim($_REQUEST['CommentField']) == ''){
+		$Error['CommentField']= "required";
+	}
+}
+
+
+function DisplayComment(){
+	$result = dbQuery("
+		SELECT * FROM BlogComments
+		ORDER BY id DESC
+	")->fetchAll();
+	return $result;
+}
+
+
+function JoinTables(){
```

Comment:
Another generic function name. When you're naming functions, imagine someone has to read your code having never seen it before, and they don't actually get to see what is inside of the functions. They should be able to tell what's going on just based on the function names, and how they're being called.